pallets/ddc-customers/src/tests.rs

```diff
@@ -0,0 +1,163 @@
+//! Tests for the module.
+
+use super::{mock::*, *};
+use ddc_primitives::ClusterId;
+
+use frame_support::{assert_noop, assert_ok};
+use pallet_balances::Error as BalancesError;
+
+#[test]
+fn create_bucket_works() {
```

Comment:
add another test to create 2 buckets and that 2 buckets can co-exist

Reply:
done